src/migrator/sync.ts

```diff
@@ -146,6 +159,10 @@ export const config: Config = {
 					ALTER TABLE "migration lock"
 					ADD COLUMN IF NOT EXISTS "modified at" TIMESTAMP DEFAULT CURRENT_TIMESTAMP NOT NULL;
 				`,
+				'14.55.0-last-execution-time': `
+					ALTER TABLE "migration status"
+					ADD COLUMN IF NOT EXISTS "last execution time ms" INTEGER NULL;
```

Comment:
This would also need to drop the `start time` column, and that potentially makes it a breaking change

Reply:
@Page- 
I was unsure how to handle it. Basically PineJS doesn't know about old instances that reference the `start time`. My non favourite solution is to keep the column as is. My preferred solution would be a 2 staged deployment. 

Basically it's not breaking the code it's breaking old-still running instances which require `start time` to be existing.
How was this handled in the past?